x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_summary/last_ten_test_runs.tsx

```diff
@@ -118,6 +118,7 @@ export const LastTenTestRuns = () => {
     },
   ];
 
+  const historyIdParam = monitor?.custom_heartbeat_id ?? monitor?.id;
```

Comment:
Suggestion
```suggestion
  const historyIdParam = monitor?.[ConfigKey.CUSTOM_HEARTBEAT_ID] ?? monitor?.id;
```

Reply:
++ I need to get used to relying on these!